zkevm-circuits/src/table/keccak_table.rs

```diff
@@ -144,3 +152,144 @@ impl KeccakTable {
         ]
     }
 }
+
+/// Keccak Table, used to verify keccak hashing from hi/lo input.
+#[derive(Clone, Debug)]
+pub struct KeccakTable2 {
```

Comment:
yes for now because I am using the hi/lo keccak table but the PI circuit had the other type. 
The reason I didn't merge them is because I was having big trouble with this refactoring and wanted to simplify to the absolutely necessary first. There is an open issue to address this later on https://github.com/taikoxyz/zkevm-circuits/issues/148, but for now the priority is to merge to master.